app/views/miq_request/_request_logs.html.haml

```diff
@@ -0,0 +1,31 @@
+.col-md-12
+	.col-md-4
+		%ul.nav.nav.tabs{:role => "tablist",:id=> "myTabs"}
+			- RequestLogs.select('request_logs.object_type').where(:miq_requests_id => @miq_request.id).group(:object_type).each do |task|
```

Comment:
I think this can be simplified to 

```suggestion
			- @miq_request.request_logs.group(:object_type).each do |task|
```

Reply:
Should there be some ordering to the grouped object_types?  As written, this will return things in a random order.